### DISCUSSION TITLE
Portal annotations

### DISCUSSION THREAD

**Discussion by aalbaali** [ahrm/sioyek#490] [Q&A] [answered]
First of all, thank you for this great application! It's been awesome!

I've been getting accustomed to portals and how to use them. One thing I noticed about them is that after creating a portal, there's no indication that a portal exists on that page/line. That is, there's a good chance that I'd come to the page a few weeks after creating a portal, and then completely forget that a portal was created.

I'm wondering if there's a custom setting that allows the annotation of portals after creating them. For example, colour the lines like in hyperlinks, or have a post-it note symbol on the side, etc. This would tell the user (i.e., me in few weeks) that there's a portal on this page/link to some other location.

**Comment by ahrm** [comment, marked as answer]
Added a `show_close_portal_in_statusbar` config option in 34d474e488f4a7f174c4d504434647e43f0e9372. If it is set, it will show a message in statusbar if there is a nearby portal like this:
![portals](https://user-images.githubusercontent.com/6392321/201960203-3ac8c328-9641-4696-b46c-e0f28febcef1.png)

**Reply by aalbaali** [reply]
Awesome! Thanks for the feature!
